src/Routed/SingleReviewPage.jsx

```diff
@@ -64,10 +109,29 @@ export default function SingleReviewPage() {
             {isLoading ? (
```

Comment:
Line 106 -> the way we render the number of comments isn't dynamic e.g. when I post a new comment it doesn't change unless I refresh but I'd way prefer to see this number change when I post or delete straight away